cf-agent/cf-agent.c

```diff
@@ -2103,13 +2103,13 @@ static int NoteBundleCompliance(const Bundle *bundle, int save_pr_kept, int save
     }
     else
     {
-        Log(LOG_LEVEL_VERBOSE, "A: Promises kept in '%s' = %.0lf", bundle->name, delta_pr_kept);
-        Log(LOG_LEVEL_VERBOSE, "A: Promises not kept in '%s' = %.0lf", bundle->name, delta_pr_notkept);
-        Log(LOG_LEVEL_VERBOSE, "A: Promises repaired in '%s' = %.0lf", bundle->name, delta_pr_repaired);
+        Log(LOG_LEVEL_INFO, "A: Promises kept in '%s' = %.0lf", bundle->name, delta_pr_kept);
+        Log(LOG_LEVEL_INFO, "A: Promises not kept in '%s' = %.0lf", bundle->name, delta_pr_notkept);
+        Log(LOG_LEVEL_INFO, "A: Promises repaired in '%s' = %.0lf", bundle->name, delta_pr_repaired);
```

Comment:
Promises repaired makes /some/ sense in Inform, but the specific changes should already be in the inform outut.

Reply:
I don't think it's consistently reported, and per bundle is very useful.